Add Apereo header checking to build.gradle (#7)
apereo/openEQUELLA-admin-console-package: README.md

```
~$ ./gradlew :configurator:run --console=verbose
```

##Apereo Header validation

> A task has been added that will check that each source file in the common and configurator projects has an Apereo 

Reviewer: ```suggestion
The gradle build includes a task which will check that each source file in the common and configurator projects has an Apereo 
```

license notice at the top of the file. This license is stored in `LICENSE/LICENSE` in the root of this repository.
In order to apply the headers, run:

```
-$ ./gradlew licenseFormat
```
 In order to check that the headers have been applied, run:
 
 ```
 -$ ./gradlew license
 ```

>  

Reviewer: ```suggestion
NOTE: The license check is executed as part of the standard `build` target.
```
